debezium-connector-mysql/src/main/java/io/debezium/connector/mysql/MySqlConnectorConfig.java

```diff
@@ -508,6 +508,14 @@ public static EventProcessingFailureHandlingMode parse(String value) {
                                                          + "Each distinct MySQL installation should have a separate namespace and monitored by "
                                                          + "at most one Debezium connector.");
 
+    public static final Field ON_CONNECT_STATEMENTS = Field.create("database.initial.statements")
+                                                           .withDisplayName("Initial statements")
+                                                           .withType(Type.STRING)
+                                                           .withWidth(Width.LONG)
+                                                           .withImportance(Importance.LOW)
+                                                           .withDescription("A semicolon separated list of SQL statements to be executed when connection to database is established. "
```

Comment:
I think we should clarify whether that only applies to snapshot reading, log reading, or both.

Reply:
Also the default (no statements) should be mentioned.